Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -9,8 +9,14 @@ import UIKit
 
 class AnimeViewController: UIViewController {
     
+    
+    
     var searchTask: DispatchWorkItem?
     private var animeViewModel = AnimeViewModel()
+    private var page = 10
```

Comment:
Why is page set to 10, what number does it start from in programming? Tobi you need to start paying more attention this is laziness...Can you start reviewing your work properly please cos this is wasting time.